clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1121,7 +1121,7 @@ public void testNormalJoinGroupFollower() {
                     sync.groupAssignments().isEmpty();
         }, syncGroupResponse(assigned, Errors.NONE));
 
-        coordinator.joinGroupIfNeeded(time.timer(Long.MAX_VALUE));
+        coordinator.joinGroupIfNeeded(time.timer(Long.MAX_VALUE),null);
```

Comment:
We should pass in a reasonable timer as the 2nd parameter (just like the 1st parameter). Same comments to below.

Reply:
change codes already commit